server/plugin.go

```diff
@@ -60,14 +60,28 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 			return false
 		}
 
-		if textNode, ok := node.(*markdown.Text); ok {
-			startPos, endPos := textNode.Range.Position+offset, textNode.Range.End+offset
-			origText := postText[startPos:endPos]
+		origText := ""
+		startPos := 0
+		endPos := 0
+
+		if autolinkNode, ok := node.(*markdown.Autolink); ok {
+			startPos, endPos = autolinkNode.RawDestination.Position+offset, autolinkNode.RawDestination.End+offset
+
```

Comment:
i'd remove this linebreak or add it in the `else if` block as well for the sake of consistency